Modules/_heapqmodule.c

```diff
@@ -450,48 +487,68 @@ siftup_max(PyListObject *heap, Py_ssize_t pos)
     return siftdown_max(heap, startpos, pos);
 }
 
+
+/*[clinic input]
+_heapq._heappop_max
+
+    heap: object
+    /
+
+Maxheap variant of heappop
```

Comment:
Why was the period at the end of the doc-string removed?

This applies here and in the other max-heap-variant functions.

Reply:
Good catch and thank you @taleinat !